internal/deviations/deviations.go

```diff
@@ -89,6 +89,11 @@ func GRIBIMACOverrideWithStaticARP(_ *ondatra.DUTDevice) bool {
 	return *gribiMACOverrideWithStaticARP
 }
 
+// CliTakesPrecedenceOverOC returns whether config pushed through origin CLI takes precedence over config pushed through origin OC.
+func CliTakesPrecedenceOverOC(_ *ondatra.DUTDevice) bool {
```

Comment:
Sorry, I missed the initialism here. "CLI" should be in all caps here and in the comment right above:
https://google.github.io/styleguide/go/decisions#initialisms

Reply:
Done